src/env.ts

```diff
@@ -15,6 +15,7 @@ import nodeCompatPreset from "./presets/nodeless";
  */
 export function defineEnv(opts: CreateEnvOptions = {}): {
   env: Environment;
+  presets: Preset[];
```

Comment:
nit: update l10 above with `presets`

Reply:
Not sure to undrestand, L10 is part of example code:

<img width="517" alt="image" src="https://github.com/user-attachments/assets/61721876-10f8-4ac8-8895-8d2f74c65324" />